portfolio/src/main/java/com/google/sps/data/Post.java

```diff
@@ -0,0 +1,29 @@
+
+package com.google.sps.data;
+
+
+/**
+ * Class representing an individual post.
+ *
```

Comment:
nit. this comment can all go into 1 line

Reply:
Okay, fixed that.